db/revision.go

```diff
@@ -323,7 +323,7 @@ func (db *Database) refreshPreviousRevisionBackup(ctx context.Context, docid str
 }
 
 // Currently only used by unit tests - deletes an archived old revision from the database
-func (db *Database) PurgeOldRevisionJSON(ctx context.Context, docid string, revid string) error {
+func (db *DatabaseCollection) PurgeOldRevisionJSON(ctx context.Context, docid string, revid string) error {
```

Comment:
I was going to say this should be DatabaseCollectionWithUser, but looking at the usage I think it's fine the way it is.